# T-BAS v2.1 Tutorial 1: Viewing Trees, Alignments, and Specimen Metadata

1. In this tutorial we will be using the *Ramularia* reference tree as an example to illustrate the features available when viewing trees in T-BAS. To begin, go to the T-BAS start page (https://tbas.hpc.ncsu.edu/start) and click on the **T-BAS Trees** button (see arrow).
![](images/tbas-tutorial/tutorial1.1.png)
2. Click on the blue bullet to select the *Ramularia* reference tree.  Note only trees with a solid blue bullet in the guide tree contain data.
![](images/tbas-tutorial/tutorial1.2.png)
3. The *Ramularia* tree is now highlighted in red.  Below the guide tree are options (blue buttons) for **View Tree Data** (i.e., viewing/downloading of tree, alignments and specimen metadata), and **Place Unknowns** (i.e., phylogenetic placement of unknown sequences).
![](images/tbas-tutorial/tutorial1.3.png)
4. Clicking on the **View Tree Data** button will open up a new tab with the *Ramularia* tree. At the top of the left side pane are buttons for **help** (tutorials), **new data** (to place unknown sequences on tree), **update voucher** (to upload new specimen metadata), **color editor** (to assign new colors to attributes in tree), **reset defaults** (to restore default settings; if your settings do not show the defaults shown in the screenshots in this tutorial, hit reset) and **select by list** (to upload a file with a list of taxa to highlight in tree). Entering text in the **Search** box will search for the name across all tree data.  The options under **Highlighted taxa** are available when user selects a leaf or clade in the tree or taxa are selected from an uploaded list.  Note the default settings for **Layout** (radial), **Format** (FASTA - aligned), and **Sequence** (RPB2) in this example.  There are subheadings under **Sequence** if the user wants to view each locus **separate** or multiple loci **concatenated**.
![](images/tbas-tutorial/tutorial1.4.png)
5. Scroll to the bottom of the left side pane to see additional options. Under the Sequence heading and immediately below the separate and concatenated alignment options is a pull down for Show taxa for data (to highlight taxa for different locus combinations) and below the pull down are the alignment default viewing options: remove taxa with no data, remove sites with all gaps, and remove unalignable regions; to view more or less alignment data, check or uncheck one or more of these boxes. The Display section provides options to colorize all leaves, colorize excluding singletons and colorize only singletons. Legends would normally appear in the right-side pane in a single column if Show 1 column legend is checked (e.g. see step 6); unchecking will display legends side by side. Colors can be shown with no transparency.
![](images/tbas-tutorial/tutorial1.5.png)
Text options include italicizing names and adjusting font size. The Colorize Leaves option will colorize the outer leaves of the tree according to specimen metadata by selecting from the pull down.  Additional outer rings (i.e. bands) can be added â€“ up to a maximum of 12 by clicking on the more+ option. Additional options will Colorize Branches and the Width of branch lines can be adjusted. Bootstrap values can be displayed on branches or by thickening the lines in the tree for a specified bootstrap threshold value. Edge numbers are used when placing unknowns in T-BAS (see Tutorial 2). Branch lengths are by default drawn to scale. Trees and legend can be Exported separately and combined as PNG, SVG, PDF, and multipage PDF. The Zoom shows the default tree magnification of 1.0, which can be adjusted by clicking the ![](images/tbas-tutorial/magnifier.jpg) {height: 30px;} icons.
